server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/MIMEMessageConverterTest.java

```diff
@@ -446,4 +446,27 @@ public void convertToMimeShouldAddAttachmentAndMultipartAlternativeWhenOneAttach
         assertThat(attachmentPart.getMimeType()).isEqualTo(expectedMimeType);
         assertThat(attachmentPart.getHeader().getField("Content-ID").getBody()).isEqualTo(expectedCID);
     }
+
+    @Test
+    public void convertShouldEncodeWhenNonASCIICharacters() {
+        // Given
+        MIMEMessageConverter sut = new MIMEMessageConverter();
+
+        CreationMessage testMessage = CreationMessage.builder()
+                .mailboxIds(ImmutableList.of("dead-bada55"))
```

Comment:
please create a builder method that doesn't force you to create a singleton list